##Parameter for collision_avoidance.cpp

##  期望位置_x;
target_x : 7.0

## 期望位置_y
target_y : -2.0

##追踪部分位置环P
p_xy : 1.4

## 单位:米
R_outside : 1.2

## 单位:米
R_inside : 0.3

## 追踪部分速度限幅
vel_track_max : 0.25

## 大圈比例系数
p_R : 0.6

## 小圈比例系数
p_r : 1

## 避障部分最大速度
vel_collision_max : 0.3

## 总速度最大速度
vel_sp_max : 0.3


## 激光雷达范围设定
range_min : 0
range_max : 359

##ABC点坐标
A_x : 2.25
A_y : 0

B_x : 0.25
B_y : -2

C_x : 2.25
C_y : -2

##flytwopoins点坐标
fA_x : 1.8
fA_y : 1

fB_x : 1.8
fB_y : -1

